helix-term/src/keymap/default.rs

```diff
@@ -311,6 +311,9 @@ pub fn default() -> HashMap<Mode, Keymap> {
 
         "C-a" => increment,
         "C-x" => decrement,
+
+        "A-a" => increasing_increment,
+        // "A-x" => decreasing_decrement,
```

Comment:
Why is this uncommented?

Reply:
`A-x` is already used for `shrink_to_line_bounds`.

Reply:
> `A-x` is already used for `shrink_to_line_bounds`.

Do you have any suggestions for a better keymap?